test/integration/operator/garden/garden/garden_suite_test.go

```diff
@@ -47,7 +47,7 @@ var (
 var _ = BeforeSuite(func() {
 	// a lot of CPU-intensive stuff is happening in this test, so to
 	// prevent flakes we have to increase the timeout here manually
-	SetDefaultEventuallyTimeout(10 * time.Second)
+	SetDefaultEventuallyTimeout(15 * time.Second)
```

Comment:
Is this change really needed?

Reply:
Unfortunately, yes.
Otherwise the test will often time out under load.